### ISSUE TITLE
Extension prevents Grammarly extension to autocorrect

### ISSUE THREAD

**Issue by g-ongenae** [hediet/browser-ext-github-monaco#31]
#### With extension
![Capture d’écran 2021-07-20 à 10 14 53](https://user-images.githubusercontent.com/9453090/126286335-280e4aa8-756e-4e30-a919-2c52bf590014.png)

#### Without extension
![Capture d’écran 2021-07-20 à 10 16 17](https://user-images.githubusercontent.com/9453090/126286338-28c571c1-4c1a-481b-bd00-53e6b424eb86.png)

**Comment by hediet** [comment]
Unfortunately, I fear grammarly and monaco (or in general every advanced text-editor) are incompatible.
At least it requires a lot of work to support grammar checkers.

**Comment by g-ongenae** [comment]
Just a rough idea, would it be possible to have a button disable, so Grammarly (or any other) would be the only one active?

Otherwise, it's doesn't matter, I'll just have to improve my spellings on my own. 😆  (Thanks for this great extension, btw. 👍)

**Comment by hediet** [comment]
> Just a rough idea, would it be possible to have a button disable, so Grammarly (or any other) would be the only one active?

Yes that could be an option. The original text area still exists.
